app/views/layouts/application.html.erb

```diff
@@ -3,4 +3,8 @@
   <%= javascript_include_tag 'application' %>
 <% end %>
 
+<% content_for :navbar_items do %>
+	<li><a href="/local-authorities">Local Authorities</a></li>
+<% end %>
```

Comment:
This probably wants to be 2-spaces, not 1-tab.  Rubocop doesn't run on erb files, so the linter won't have caught it.